machine_learning_hep/data/data_prod_20200304/database_ml_parameters_LcpK0spp_0304_jet.yml

```diff
@@ -422,8 +430,12 @@ LcpK0spp:
 
       branching_ratio: 1.09e-2
       xsection_inel: 57.8
+      lb_branching_ratio: 0.144
+      convert_const: 1.e-9 # pb -> mb
+      antipart_count: 2
```

Comment:
Add comments with explanation.
Why is `convert_const` needed as a configurable database parameter?